src/pudl/metadata/codes.py

```diff
@@ -209,15 +209,16 @@
                     "duquense",
                     "Duquense Light Company & Energy Systems Associates",
                 ),
-                ("DS", "doosan" "Doosan"),
+                ("DS", "doosan", "Doosan"),
```

Comment:
Woah, did ruff somehow catch this too?

Reply:
Yeah, it's got a "oops did you accidentally concatenate two strings implicitly?" check!